Content.Server/_Goobstation/Devil/Grip/DevilGripSystem.cs

```diff
@@ -0,0 +1,81 @@
+// SPDX-FileCopyrightText: 2025 GoobBot <uristmchands@proton.me>
+// SPDX-FileCopyrightText: 2025 Solstice <solsticeofthewinter@gmail.com>
+// SPDX-FileCopyrightText: 2025 SolsticeOfTheWinter <solsticeofthewinter@gmail.com>
+// SPDX-FileCopyrightText: 2025 TheBorzoiMustConsume <197824988+TheBorzoiMustConsume@users.noreply.github.com>
+// SPDX-FileCopyrightText: 2025 Tim <timfalken@hotmail.com>
+// SPDX-FileCopyrightText: 2025 Timfa <timfalken@hotmail.com>
+//
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+using Content.Shared._Goobstation.Devil;
+using Content.Shared._Goobstation.Religion;
+using Content.Server.Chat.Systems;
+using Content.Server.Speech.EntitySystems;
+using Content.Shared.Actions;
+using Content.Shared.Chat;
+using Content.Shared.Damage.Systems;
```

Comment:
Unused using directives
```suggestion
using Content.Shared._Goobstation.Devil;
using Content.Server.Chat.Systems;
using Content.Server.Speech.EntitySystems;
using Content.Shared.Actions;
using Content.Shared.Damage.Systems;
```

Reply:
Goobcode, leaving as is for parity